tools/xtce_generation/telemetry_generator.py

```diff
@@ -195,7 +212,9 @@ def create_parameter_types(self, parameter_type_set, unique_lengths):
 
         return parameter_type_set
 
-    def create_ccsds_packet_parameters(self, parameter_set, ccsds_parameters):
+    def create_ccsds_packet_parameters(
+        self, parameter_set: list, ccsds_parameters: list
+    ) -> list:
```

Comment:
```suggestion
    def create_ccsds_packet_parameters(
        self, parameter_set: Et.Element, ccsds_parameters: list
    ) -> Et.Element:
```

I think the fix here is similar to the `create_parameter_types()` function